audit/simpleAuditor_test.go

```diff
@@ -16,12 +16,14 @@ package audit
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"io/ioutil"
 	"os"
 	"testing"
 	"time"
 
-	"github.com/codenotary/immuproof/cnc"
+	"github.com/codenotary/immuproof/cnc/cnctest"
```

Comment:
Please group and sort imports properly. I recommend https://github.com/incu6us/goimports-reviser